lib/execution/transaction.js

```diff
@@ -78,7 +78,10 @@ class Transaction extends EventEmitter {
   }
 
   begin(conn) {
-    return this.query(conn, 'BEGIN;');
+    const setIsolationLevel = this.isolationLevel
+      ? this.query(conn, `SET ISOLATION LEVEL ${this.isolationLevel};`)
```

Comment:
Which databases would that work for? I think this will currently affect Oracle and MySQL.